department-of-veterans-affairs/abd-vro
# Mcp 2098: Add claimId to python logging

## What was the problem?

The logs from python services were difficult to attribute to individual claims. 

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]

Adds claimId to all python logs from assessment services
## How to test this PR
- Step 1 build and run app. Try endpoint for full health assessment (diagnostic code: 7101). Look at logs for python docker service and insure that claimId is there. 
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
